src/components/ui/dialog.tsx

```
@@ -21,13 +19,13 @@
   <DialogPrimitive.Overlay
     ref={ref}
     className={cn(
-      "data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 fixed inset-0 z-50 bg-black/80 data-[state=closed]:animate-out data-[state=open]:animate-in",
+      "fixed inset-0 z-50 bg-black/80  data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0",
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)